pkg/controller/controller_binding.go

```diff
@@ -149,6 +174,14 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 		return err
 	}
 
+	if binding.Status.OrphanMitigationInProgress == true {
+		err := c.doOrphanMitigation(instance, toUpdate)
+		if err != nil {
```

Comment:
Nit: can be combined into one line:
`if err := c.doOrphanMitigation(instance, toUpdate); err != nil {`